tests/pipelines/test_step_utils.py

```diff
@@ -87,12 +87,14 @@ def test_get_data_profile_truncates_large_data_frame(
     step_utils._MAX_PROFILE_CELL_SIZE = max_cells
     step_utils._MAX_PROFILE_COL_SIZE = max_cols
     step_utils._MAX_PROFILE_ROW_SIZE = max_rows
-    profile = get_pandas_data_profile(data_frame, "fake profile")
-    assert json.loads(profile.to_json())["table"]["n_var"] == expected_cols
-    assert json.loads(profile.to_json())["variables"]["0"]["count"] == expected_rows
+    with mock.patch.object(pandas_renderer, "get_html") as mock_pandas_renderer_html:
+        get_pandas_data_profile(data_frame, "fake profile")
+        truncated_df = mock_pandas_renderer_html.call_args_list[0][0][0][0][1]
```

Comment:
Wow. How deep is this dataframe buried! Can you add a comment to indicate the various levels of the call args?